docs/problems/nn.rst

```diff
@@ -11,5 +11,6 @@ Dataset
     :toctree: _autosummary
 
     dataset.create_gaussian_mixture_samplers
```

Comment:
I think I'd prefer to have shorter names, like `gaussian_mixture_sampler(s)`, can you please adjust (will need to also change the other NN notebooks).